broker/test/test_broker/acceptance/service-broker-api.catalog.spec.js

```diff
@@ -35,7 +35,8 @@ describe('service-broker-api', function () {
             name: 's1'
           }
         }]
-      });
+      },
+        {});
       mocks.apiServerEventMesh.nockGetResourcesAcrossAllNamespaces(CONST.APISERVER.RESOURCE_GROUPS.INTEROPERATOR, CONST.APISERVER.RESOURCE_TYPES.INTEROPERATOR_PLANS, {
         items: [{
```

Comment:
`mocks.apiServerEventMesh.nockGetResourcesAcrossAllNamespaces` takes a `query string` as its 4th argument. From nock v11.0.0 query string cannot be `undefined`. Passing an empty `{}` instead.